notebooks/data_analysis/pathfinder_items_other_columns.ipynb

```diff
@@ -0,0 +1,932 @@
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "items[pf2e.condition.notnull()]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "All of items with `condition: True` have type *condition*\n",
+    "\n",
+    "*Not important*"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "**Flags**:\n",
+    "* core - *not important*\n",
+    "* pf2e:\n",
+    "    * linkedWeapon - maybe important if we are going to somehow count items of monster because e.g. Gancanagh have only one rapier (but a silver one)\n",
+    "    * condition - *not important*"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Summary\n",
+    "* _id - *not important*\n",
+    "* img - *not important*\n",
+    "* name - *not important*\n",
+    "* sort - **???**\n",
+    "* type - **might be useful**\n",
```

Comment:
Tak, to jest do tego: `"coś w stylu "Melee  hand +24 [+19/+14] (finesse, magical), Damage 4d8 negative plus paralyzing touch" (to akurat wzięte z Licha, więc jest dość specyficzne)"` 

i tego `"liczba czarów poszczególnych poziomów, np. Priest of Pharasma ma "Divine Prepared Spells DC 24, attack +16; 3rd [circle of protection](https://2e.aonprd.com/Spells.aspx?ID=38), [heal](https://2e.aonprd.com/Spells.aspx?ID=148) (x3), [searing light](https://2e.aonprd.com/Spells.aspx?ID=269) (x2); 2nd [gentle repose](https://2e.aonprd.com/Spells.aspx?ID=131), [silence](https://2e.aonprd.com/Spells.aspx?ID=287), [spiritual weapon](https://2e.aonprd.com/Spells.aspx?ID=306); 1st [disrupting weapons](https://2e.aonprd.com/Spells.aspx?ID=80), [mindlink](https://2e.aonprd.com/Spells.aspx?ID=195), [spirit link](https://2e.aonprd.com/Spells.aspx?ID=302); Cantrips (3rd) [detect magic](https://2e.aonprd.com/Spells.aspx?ID=66), [disrupt undead](https://2e.aonprd.com/Spells.aspx?ID=79), [light](https://2e.aonprd.com/Spells.aspx?ID=171), [read aura](https://2e.aonprd.com/Spells.aspx?ID=246), [shield](https://2e.aonprd.com/Spells.aspx?ID=280)" - czyli 6 czarów 3 poziomu, 3 czary 2 poziomu, 3 czary 1 poziomu (kantyczki raczej można odpuścić, one są słabe)"`